next/components/mdx/mdx.tsx

```diff
@@ -289,11 +90,126 @@ export const MDXComponentPage = ({
+                                                    {prop.required && (
+                                                        <div className="ml2">
+                                                            <Tag type="required" />
+                                                        </div>
+                                                    )}
+                                                    {deprecated && (
+                                                        <div className="ml2">
+                                                            <Tag type="deprecated" />
+                                                        </div>
+                                                    )}
+                                                </div>
+                                                <div className="black-300 mw8 mb2">
+                                                    {prop.description && (
+                                                        <MDXRenderer>
+                                                            {prop.description.description}
+                                                        </MDXRenderer>
+                                                    )}
+
+                                                    {deprecated &&
+                                                        isString(deprecated.description) && (
+                                                            <p>
+                                                                <b>Note:</b>{' '}
+                                                                {deprecated.description}
+                                                            </p>
+                                                        )}
+                                                </div>
+                                                <div className="flex">
+                                                    {prop.tsType && (
+                                                        <Text
+                                                            size={2}
+                                                            className="mr4 w-50 s_w-40 m_w-30 m_w-20"
+                                                            elementName="div"
+                                                        >
+                                                            <div className="b black-300">Type</div>
+                                                            <PropType
+                                                                type={prop.tsType.name}
+                                                                value={prop.tsType.elements}
+                                                            />
+                                                        </Text>
+                                                    )}
+                                                    {get(prop.defaultValue, 'value') && (
```

Comment:
prefer `?.` to `_.get` in TS files

Reply:
I should've indicated this in the PR description, but this file was almost entirely copied over from the Gatsby code. I'd like to leave this as-is for now.